scipy/stats/stats.py

```diff
@@ -1377,10 +1377,9 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     References
     ----------
     .. [1] D'Agostino, R. B. (1971), "An omnibus test of normality for
-           moderate and large sample size," Biometrika, 58, 341-348
+           moderate and large sample size", Biometrika, 58, 341-348
 
-    .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Testing for
-           departures from normality," Biometrika, 60, 613-622
+    .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Tests for departure from normality. Empirical results for the distributions of b2 and ...", Biometrika, 60, 613-622
```

Comment:
Something went wrong with the end of the line, this `...` is not right --- however I've not idea what the symbol in the reference means though. 

The long lines should be wrapped to <80 chars please.

Maybe the best thing to do is to correct the first sentence of the title (Testing->Tests) and be done with it.